iron-selectable.html

```diff
@@ -148,7 +149,7 @@
       this._observer = this._observeItems(this);
       this._updateItems();
       if (!this._shouldUpdateSelection) {
-        this._updateSelected(this.attrForSelected,this.selected)
+        this._updateSelected(this.selected);
```

Comment:
jup, that's right! I feel always uncomfortable with the observation definition, where you use your argument to trigger the function execution knowing you can always access the property inside your element... :)